src/ampycloud/data.py

```diff
@@ -406,7 +406,7 @@ def metarize(self, which: int = 'slices', base_lvl_alt_perc: float = 5.,
             elif (self.max_hits_per_layer - pdf.iloc[ind]['n_hits']) <= lim8:
                 pdf.iloc[ind]['okta'] = 8
             else:
-                pdf.iloc[ind]['okta'] = int(wmo.perc2okta(pdf.iloc[ind]['perc']))
+                pdf.iloc[ind]['okta'] = int(wmo.perc2okta(pdf.iloc[ind]['perc'])[0])
```

Comment:
Fair point. If we do this change, we should probably try to do it "everywhere". Let me know what you find out, and we can see about implementing it.

Reply:
the indexing thing somehow caught my interest as I often do indexing operations on DataFrames and never really cared about whether I do the right thing. So I calculated some timings (apologies for the bad plots) and it seems that the `get_loc` way is indeed faster.
![image](https://github.com/MeteoSwiss/ampycloud/assets/26576213/7d6ef561-7c5a-481b-85e7-297ef687d452)

Reply:
Thanks for the detailed analysis @regDaniel ! I've incorporated the suggested syntax change - it's better to call these in one go instead of a dangerous chain. Turns out, this was not present in many places in the code, so it was fairly easy to address.